### ISSUE TITLE
Choose to have no day selected on initial render?

### ISSUE THREAD

**Issue by BillyFigueroa** [wix/react-native-calendars#310]
Please make our job easier by filling this template out to completion. If you're requesting a feature instead of reporting a bug, please feel free to skip the Environment and Reproducible Demo sections.

## Description

Is there a way to turn off having a selected day while still being able to have it highlight a day when you do select one?

## Expected Behavior

I would like to be able to not have ANY day selected as a default and force the user to select a date. There are some things I want to grab when a user clicks on a date and I rather do it off a user click rather than using some loading event

## Observed Behavior

Currently if I do not specify a selected date, the current date is selected. There are 2 issues with this in my case

1) I do not want any date selected by default
2) I am using disabledByDefault option so I can then decide which dates can be enabled by using

```
[date]: { disabled: false }
```

If the current date (as in Today) is NOT a date I have decided is active or clickable, it should not be highlighted

Is there a way to get this option to have something like selected="none" or if you pass null it doesn't pick the current day?

## Environment

This is on version v1.16.1

**Comment by tautvilas** [comment]
Hey, I'm not sure I get your problem. If you checked Calendars example there is a calendar that does not have date selected on initial load. Or are you talking about agenda component? Could you add screenshot?

**Comment by BillyFigueroa** [comment]
Yes sorry for not being clear. I am using the Agenda. I am having to hide the content area because what I really want is JUST THE STRIP portion. Is there a way to not have a day selected? I have this working now where I just use some of the event functions it provides to load data for the day it picks as default but I would prefer to allow the user to select a day with out it defaulting to one

**Comment by mjhassan** [comment]
@tautvilas, any update on this enhancement?

**Comment by tautvilas** [comment]
Unfortunately no

**Comment by scblason** [comment]
I'm using the ExpandableCalendar and I'm facing a similar issue where I don't set the "date" prop in the CalendarProvider and it always shows up as selected. Is there any workaround for this?
